### PR TITLE
Address HCAL review comments regarding CondFormats/HcalObjects

### PR DESCRIPTION
**Pull request by mariadalfonso**
as part of review of
https://github.com/cms-sw/cmssw/pull/31720

### CODE REVIEW

**Comment by abdoulline** [comment]
Thank you, Maria!
Also would like to explicitly add Igor @igv4321

**Comment by fwyzard** [comment]
:construction: Validation running at fu-c2a02-37-01:/data/user/fwyzard/patatrack/validation/run_565.IkPkbp0Q8A ...

**Comment by fwyzard** [comment]
No changes in the HCAL (or other) workflows, as expected.

File: CUDADataFormats/HcalDigi/src/classes_def.xml [hunk 1]
```diff
@@ -2,51 +2,35 @@
     <class name="hcal::DigiCollectionBase<calo::common::VecStoragePolicy<std::allocator>>" />
     <class name="hcal::DigiCollectionBase<calo::common::VecStoragePolicy<calo::common::CUDAHostAllocatorAlias>>" />
 
-    <class name="hcal::DigiCollection<hcal::Flavor01, calo::common::VecStoragePolicy<std::allocator>>" />
-    <class name="hcal::DigiCollection<hcal::Flavor2, calo::common::VecStoragePolicy<std::allocator>>" />
```
Comment (fwyzard) [inline CUDADataFormats/HcalDigi/src/classes_def.xml:6]: @mariadalfonso should `Flavor2` and `Flavor4` also be drmoved from CUDADataFormats/HcalDigi/interface/DigiCollection.h ?

Comment (mariadalfonso) [reply CUDADataFormats/HcalDigi/src/classes_def.xml:6]: correct, they can be removed as well 

https://github.com/cms-sw/cmssw/pull/31720/files#diff-37c00e58e6708c06d66be199b3574bb7d2bfd6307d919fdd941eb29102655d1bR23

https://github.com/cms-sw/cmssw/pull/31720/files#diff-37c00e58e6708c06d66be199b3574bb7d2bfd6307d919fdd941eb29102655d1bR45

will you do , or shall I ?

**Comment by fwyzard** [comment]
I will do it for the Patatrack PR. Can you do it for the one for CMSSW ?

**Comment by mariadalfonso** [comment]
> I will do it for the Patatrack PR. Can you do it for the one for CMSSW ?

uhm, the only PR I own (13/N) contains only the conditions for the CondFormats/HcalObjects, 
I didn't push there the files CUDADataFormats/HcalDigi

so both the patatrack and cms-sw (8/N) are yours

**Comment by fwyzard** [comment]
Ah, perfect, then I'll propagate the change myself with the next update.

**Pull request merged**